src/App.js

```diff
@@ -1,13 +1,72 @@
 import React from 'react';
 import './App.css';
+import Button from "./Components/Button";
+import Product from "./Components/Product";
+import Tile from "./Components/Tile";
+import bag1 from "./assets/bag_1.png"
+import bag2 from "./assets/bag_2.png"
+import bag3 from "./assets/bag_3.png"
+import bag4 from "./assets/bag_4.png"
+
+
 
 function App() {
-  return (
-      <h1>Handbags & Purses</h1>
-  );
+    return (
+        <>
+            <h1>Handbags & Purses</h1>
+            <nav>
+                <Button nameButton="To the collection" disabled={false}/>
```

Comment:
Je methode stemt als ik het goed heb overeen met hetgeen we van Elwyn hebben geleerd. Het antwoord model hanteert code met een clickHandler, het betreft de volgende code: <Button isDisabled={false} clickHandler={() => console.log("To the collection!")}>
          To the collection
        </Button>